.github/scripts/test.py

```
@@ -117,6 +118,14 @@ def print_nanomq_log():
     vul_test()
     print("vul_test test end")
 
+    print("issue_2246 test start")
+    if False == issue_2246_test():
```

Ruff (0.15.6)

[error] Avoid equality comparisons to `False`; use `not issue_2246_test():` for false checks

Replace with `not issue_2246_test()`

(E712)